SCClassLibrary/Common/Control/Server.sc

```diff
@@ -38,8 +40,6 @@ ServerOptions {
 	var <>threads = nil; // for supernova
 	var <>useSystemClock = false;  // for supernova
 
-	var <numPrivateAudioBusChannels=112;
```

Comment:
Yes, I worried, too. I've had a look and found no reference. Actually I am not sure if that still applies at all - I don't see where it should.